client/router_create.go

```diff
@@ -285,6 +290,10 @@ func (cli *VanClient) GetVanControllerSpec(options types.SiteConfigSpec, van *ty
 			Port:     types.ClaimRedemptionPort,
 		})
 		if options.IsIngressRoute() {
+			host := options.GetRouterIngressHost()
```

Comment:
Here again, should this be GetControllerIngressHost()? Likely in most cases these will be the same. Not even sure if we need or want to have separate options TBH.

Reply:
I went back and forth but agree it makes sense to allow separation for now. Will update.